src/wasm-lib/kcl/tests/artifact_graph_sketch_on_face_etc/artifact_graph_mind_map.snap.md

```diff
@@ -7,186 +7,6 @@ mindmap
           Wall
           SweepEdge Opposite
           SweepEdge Adjacent
-        Segment
```

Comment:
This seems very suspect, not sure how I managed to do this, but everything on my end seems to work I think, tests pass!?

Reply:
cc @jtran

Reply:
After examining the changes in the flow chart, I think this is okay. The way the mind map works is that it does a DFS traversal starting from each plane. But since the sketch-on-face is no longer on the plane, but correctly on the end-cap, it doesn't see these segments and all the extrusions from them. Does that make sense?

I'm afraid that these mind maps might be more confusing than helpful. I'm considering deleting them all. Leave a comment if you have thoughts.

Reply:
That does make sense. Not sure about if they're more confusing than helpful, I guess I didn't really know how they work.

Reply:
- https://github.com/KittyCAD/modeling-app/pull/5349